hudi-flink/src/main/java/org/apache/hudi/operator/InstantGenerateOperator.java

```diff
@@ -297,4 +297,16 @@ private void cleanMarkerDir(Path instantMarkerFolder) throws IOException {
       fs.delete(fileStatus.getPath(), true);
     }
   }
+
+  private Path generateCurrentMakerPath() {
+    String baseDir = cfg.targetBasePath.endsWith("/")
```

Comment:
Do you check if hadoop `new Path(xxx, xxx)` could handle the slash intelligently?